go/greeter/.devcontainer/devcontainer.json

```diff
@@ -1,7 +1,7 @@
 // For format details, see https://aka.ms/devcontainer.json. For config options, see the
 // README at: https://github.com/devcontainers/templates/tree/main/src/go
 {
-	"name": "Go Greeter",
+	"name": "Go 1.19",
```

Comment:
Shall we keep the original name `Go Greeter` here?
Since this is directly showing in the VSCode Title bar, having the sample name is useful